drivers/backup/auth.go

```diff
@@ -29,7 +29,8 @@ const (
 	// PxCentralAdminSecretNamespace namespace of PxCentralAdminSecretName
 	PxCentralAdminSecretNamespace = "px-backup"
 	// keycloakEndPoint Endpoint for keycloak
-	keycloakEndPoint = "pxcentral-keycloak-http:80"
+	// TODO: make the namespace configurable for service
+	keycloakEndPoint = "pxcentral-keycloak-http.px-backup.svc.cluster.local:80"
```

Comment:
For now it is ok, but if the namespace name changes the endpoint will change . Also to access the endpoint via px-central-endpoint/auth is much better as there will be no dependancy on which ns the test runs.

Lets create a separe ticket for this.